checkers/Board.py

```diff
@@ -98,14 +125,36 @@ def move_piece(self, mouse_x, mouse_y):
             self.grid[self.selected_piece[0]][self.selected_piece[1]].selected = False
             self.grid[selected_row][selected_col] = self.grid[self.selected_piece[0]][self.selected_piece[1]]
             self.grid[self.selected_piece[0]][self.selected_piece[1]] = 0
+
+
+            # Remove eaten Piece
+            sides = ['left', 'right']
+            for side in sides:
+                if side in self.opponent_piece_eaten:
+                    if (selected_row == self.opponent_piece_eaten[side][2]) & (selected_col == self.opponent_piece_eaten[side][3]):
+                        print(f"Eating {side} Side of piece.")
+                        self.grid[self.opponent_piece_eaten[side][0]][self.opponent_piece_eaten[side][1]] = 0
+            self.opponent_piece_eaten = {}
```

Comment:
This line also should put inside the _else_ statement right? Please check it @aungpaing98